core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -0,0 +1,378 @@
+package cats
+package data
+
+import NonEmptyLazyList.create
+import kernel.PartialOrder
+import instances.lazyList._
+
+import scala.collection.immutable.TreeSet
+
+object NonEmptyLazyList extends NonEmptyLazyListInstances {
+  private[data] type Base
+  private[data] trait Tag extends Any
+  type Type[+A] <: Base with Tag
+
+  private[cats] def create[A](s: LazyList[A]): Type[A] =
```

Comment:
Do you happen to know if these aren't `private[data]` just because of the tests? It seems like e.g. `NonEmptyStreamSuite` should be in `cats.data`, not `cats.tests`.

Reply:
Right now all tests suites are under `cats.tests`, do we want to make it more conventional (i.e. parallel package tree as in src)?

Reply:
I'd say yes, especially if it's the only reason these aren't more restricted. That's pretty far from the topic of this PR, though. I can take a look this weekend.

Reply:
just found out that the tests don't use them. So it's probably a simple artifact of copy paste code. I just changed all of such methods to be private[data]

Reply:
Nice, thanks!